Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:latest
```

Comment:
Always use a specific version when specifying Docker images; never use `latest`. Otherwise, upstream changes may break our Docker images.

Also, `latest` doesn't actually work the way most people initially expect it to. It's just a regular tag, so it may not actually even be the latest tag pushed.

Reply:
OK, do we have a preferred version? If not, should I go with `python:3.8.1` or its `slim` version? Tested and it works with `python:3.8.1-slim`.

Reply:
That should be fine.